Dockerfile

```diff
@@ -218,8 +223,11 @@ COPY bin bin
 COPY data data
 
 USER root
-RUN chown -R ubuntu:ubuntu /usr/src/app
-USER ubuntu
+RUN chown -R ubuntu:0 ${APP_ROOT} ${HOME} && \
+  chmod -R g=u ${APP_ROOT} ${HOME} /etc/passwd
```

Comment:
Give the user `ubuntu` and the group `root` ownership and equal access to the application directory, home directory and /etc/passwd.

Reply:
So although the Renovate image will default to run as the `ubuntu` image, when it's run in openshift it will actually run as a different user?

Reply:
Yes. By default OpenShift ignores which user is specified and replaces it with a random non-root user.
>By default, OpenShift Container Platform runs containers using an arbitrarily assigned user ID. This provides additional security against processes escaping the container due to a container engine vulnerability and thereby achieving escalated permissions on the host node.
>
>For an image to support running as an arbitrary user, directories and files that may be written to by processes in the image should be owned by the root group and be read/writable by that group. Files to be executed should also have group execute permissions.

Reply:
I think I will try to remove the `/etc/passwd` part. I needed it at some point during testing, but  things seem to work without it at the moment.
Will remove it and test it on OpenShift tomorrow.

Reply:
> Will remove it and test it on OpenShift tomorrow.
Tested without `/etc/passwd` on OpenShift, and it works just fine.